src/dm-screen/components/StatusAvatar/StatusAvatar.css

```diff
@@ -0,0 +1,12 @@
+.status-avatar {
+  align-items: center;
```

Comment:
Nitpick, but i'm curious how it looks zoomed in/out with the keyboard. I'm pretty confident it will still look good because of flex and justify content center, but just want to double check with you